app/web/src/components/RecipeItem.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import RecipeItem from './RecipeItem';
+
+describe('RecipeItem', () => {
+  const recipe = {
+    name: 'Test Recipe',
+    description: 'Test Description',
+    ingredients: [
+      { name: 'Ingredient 1' },
+      { name: 'Ingredient 2' },
+      { name: 'Ingredient 3' },
+    ],
+  };
+
+  test('renders recipe name', () => {
+    render(<RecipeItem {...recipe} />);
+    const recipeNameElement = screen.getByText('Test Recipe');
```

Comment:
Ah you're using screen here 👍

Reply:
you see that I was not consistent on the strategies!😅